### ISSUE TITLE
Spam addin dialog is squished when viewed in Safari.

### ISSUE THREAD

**Issue by glr0221** [OfficeDev/office-js#4561]
# Provide required information needed to triage your issue
Using safari browser and outlook on web, clicking on the spam addin button shows a squished dialog box.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: MAC Safari-browser Outlook/Office on the web
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: ______
* Operating System: ______
* Browser (if using Office on the web): ______
 
## Expected behavior

Dialog sizing should be consistent across all platforms. 

## Current behavior

Spam addin dialog is squished.

## Steps to reproduce

1. Download the sample code in : https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-spam-reporting
2. Sideload manifest.xml
3. Open Safari and login to your Outlook account.
4. Select an email and click on the spam addin button.
5. It looks like the one below : 
<img width="945" alt="Screenshot 2024-06-10 at 10 05 28 PM" src="https://github.com/OfficeDev/office-js/assets/3444844/db520d5d-8430-4b43-8cb4-c86a94aadb2b">

## Link to live example(s)

NA

# Provide additional details

NA

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by mobisw-msft** [comment]
To report issues with Office Add-in samples (edited) please use the link [here](https://github.com/OfficeDev/Office-Add-in-samples/issues/new/choose). If you're using the dialog API in your own add-in, you can set the width or height as a percentage of the viewport area, as [mentioned here](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/dialog-api-in-office-add-ins). This behaviour is consistent across all browsers, including Safari.

**Comment by glr0221** [comment]
@mobisw-msft i don't think this is an issue on the addin sample. This is an issue for spam addin itself. That is why I posted it here. As for the size of the viewport, yes you are right that this can be adjusted if it were an iframe opened via [displayDialogAsync](https://learn.microsoft.com/en-us/javascript/api/office/office.ui?view=common-js-preview#office-office-ui-displaydialogasync-member(1)). But unfortunately, you cannot control the size of the viewport when opening a spam addin. Thanks.

**Comment by mobisw-msft** [comment]
Hey @glr0221 , apologies for the confusion. Yes- this issue seems reproducible. @glr0221 ,

Thank you for reporting this issue regarding the Spam add-in. We have added this bug to our backlog. However, we have no timelines to share at this point.

Internal Tracking ID: 4655641
